### PR TITLE
FOUR-9361 apply media quieries to container

### PR DESCRIPTION
**Pull request by devmiguelangel**
## Issue & Reproduction Steps
1. Create screen 
2. Add line input
3. Add CSS with the following code
```css
@media screen and (max-width: 960px) {
  input {
    background-color: blue;
  }
}
```
4. Press Preview button
5. Press “Preview Web” button
6. Very the size of the screen in Web mode
7. Press “Preview Mobile“ button
8. Very the size of the screen in Web mode

Expected behavior: 
The screen size should be different for Web and Mobile, So from Media Queries, it should be applied when switching from web to mobile in preview like devtools

Actual behavior: 
The screen size is the same for Web and Mobile 

## Solution
- Apply media queries only to the container not to entire page

## How to Test
Test the steps above

## Related Tickets & Packages
[FOUR-9361](https://processmaker.atlassian.net/browse/FOUR-9361)

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

<sup> Created from VS Code using [CodeStream](https://codestream.com/?utm_source=cs&utm_medium=pr&utm_campaign=github*com)</sup>

[FOUR-9361]: https://processmaker.atlassian.net/browse/FOUR-9361?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/vue-form-renderer.vue [hunk 1]
```diff
@@ -156,14 +158,18 @@ export default {
   },
   created() {
     this.parseCss = _.debounce(this.parseCss, 500, {leading: true});
+
+    this.containerObserver = new ResizeObserver(this.onContainerObserver);
   },
   mounted() {
-    this.parseCss();
+    // this.parseCss();
```
Comment (josechirivella) [inline src/components/vue-form-renderer.vue:165]: remove

File: src/components/vue-form-renderer.vue [hunk 2]
```diff
@@ -1,6 +1,7 @@
 <template>
   <div
     id="vue-form-renderer"
+    ref="vueFormRenderer1"
```
Comment (josechirivella) [inline src/components/vue-form-renderer.vue:4]: give it another name without the number

**Review by josechirivella — CHANGES_REQUESTED** [review]

File: src/components/vue-form-renderer.vue (hunk 1 above)
Comment (devmiguelangel) [reply src/components/vue-form-renderer.vue:165]: it will be removed

File: src/components/vue-form-renderer.vue (hunk 2 above)
Comment (devmiguelangel) [reply src/components/vue-form-renderer.vue:4]: It will be renamed

**Review by ana-mauricio — CHANGES_REQUESTED** [review]
The screen is the same to web and mobile, the condition is not correct. As show in the folowing video:

https://github.com/ProcessMaker/screen-builder/assets/21198450/4e048998-a5ea-4fb4-8ceb-7adfe6ecd2d4
@devmiguelangel

**Review by josechirivella — APPROVED** [review]

**Review by ana-mauricio — APPROVED** [review]
The fix is correct
@devmiguelangel @danloa

**Pull request merged**